pkg/cri/sbserver/sandbox_status.go

```diff
@@ -57,10 +57,12 @@ func (c *criService) PodSandboxStatus(ctx context.Context, r *runtime.PodSandbox
 		status.CreatedAt = sandboxInfo.CreatedAt.UnixNano()
 	}
 
-	return &runtime.PodSandboxStatusResponse{
-		Status: status,
-		Info:   cstatus.Info,
-	}, nil
+	resp := &runtime.PodSandboxStatusResponse{Status: status}
+	if r.GetVerbose() {
+		resp.Info = cstatus.Info
```

Comment:
Unless I'm missing something, I think the idea was that we pass `r.GetVerbose()` to `controller.Status`, so its controller's responsibility to populate `Info` field depending on verbose flag.
We can keep this extra check here just in case though.

Reply:
Oh duh LOL. Okay this should be fine then, my local containerd is a lil too hacked up atm I think.. The shim_mode sandbox route however doesn't return Info https://github.com/containerd/containerd/blob/main/plugins/sandbox/controller.go#L269-L275 which seems like we should fix